code/modules/research/techweb/_techweb_node.dm

```diff
@@ -17,6 +17,7 @@
 	var/category = "Misc"				//Category
 	var/ui_x = 805 // It's location - override this in techweb_layout.dm
 	var/ui_y = 805
+	var/list/approved_jobs = list("Scientist", "Roboticist", "Research Director", "Captain", "AI", "Cyborg")
```

Comment:
Not a fan of using a list of strings like this, should piggyback off the GLOBs of positions so we’re sure they get updated in case we add a new science job. (You forgot network admin btw)

Reply:
Who

Reply:
Can you direct me towards this global that stores departmental positions if it exists